packages/engine/README.md

```diff
@@ -134,6 +135,36 @@ The CLI comes with a short help page: `cli help` or `cli -h`. A more detailed ex
 
 If one of the environment variables shown in the help page is passed, it will overwrite the default values. Parameters take precedence over environment variables.
 
+### Run a simulation from [hCore]
+
+> **Warning** - Python and Rust runners are currently not supported. Make sure your project only contains JSON and Javascript.
+
+In order to run a simulation from hCore, you can download it from the File menu and save by clicking on `Export Project`. Also see the [online documentation for hCloud][docs]. Save and unzip your project to a location of your choice, for example by
+
+```shell
+$ unzip my-project.zip -d my-hash-project
+```
+
+To run the simulation, [build the binaries](#project-setup--building) and pass your project to the CLI:
+
+```shell
```

Comment:
I think we should probably get rid of this one for now, or expand it to say to 
* run a `cargo build --release` first (which I appreciate I have suggested we remove above)
* Run the cli with the appropriate path into the target folder (which might be a bit hard to explain). We can't just say CD as relative paths are used all over the place atm. I don't think `cli` will work unless you do `./target/release/cli` but maybe I'm wrong?

Reply:
> maybe I'm wrong?

No. As currently the user have to build the binary anyway, any documentation should just user `cargo run --bin cli --`